apps/examples/elf/micomapp/Makefile

```diff
@@ -107,3 +107,13 @@ clean:
 install:
 	$(Q) mkdir -p $(SYSTEM_BIN_DIR)
 	$(Q) install $(BIN) $(SYSTEM_BIN_DIR)/$(BIN)
+
+verify:
+ifneq ($(BIN_CHECK),0)
+	@echo "Undefined Symbols"
+	$(Q) $(NM) -u -l $(BIN)
+	$(call DELFILE, $(SYSTEM_BIN_DIR)/$(BIN))
```

Comment:
Is it necessary to delete the bin file here?
We could just return after printing the error message. 
It is up to the user to perform clean. 
The elf file might be required by the user to check and fix the error messages.

Reply:
Since we are verifying it after the linking, object and binary would be generated already. If  developer/user build again without resolving undefined error, build will pass and there wont be any build error. To avoid it we delete object/binary if there is undefined symbols.

Reply:
ok